### DISCUSSION TITLE
功能问题

### DISCUSSION THREAD

**Discussion by Xiaoprinceprogrammer** [Mrs4s/go-cqhttp#1612] [Ideas]
我就是看cq功能不是很全，所以我希望官方能多加一些功能，比如QQPC登录 等等

**Comment by Ink-33** [comment]
pc登录没法做

**Reply by Xiaoprinceprogrammer** [reply]
为什么呢

**Reply by Xiaoprinceprogrammer** [reply]
> pc登录无法做

兄弟你是cq作者吧  我们加个好友可以吗 方便沟通

**Reply by Xiaoprinceprogrammer** [reply]
还有一些问题 我要和你反馈一下 加个好友吧

**Reply by Ink-33** [reply]
请将您遇到的问题通过电子邮件发到
ink33@smlk.org
感谢

**Comment by Xiaoprinceprogrammer** [comment]
为什么

**Comment by DiheChen** [comment]
![MW4{@A9@DW66V~XZ ZQLTNB](https://user-images.githubusercontent.com/52123645/182561106-03873d6e-ff03-40f2-93cf-593b8c18308c.PNG)

**Comment by synodriver** [comment]
PC登录是另一种了。。。很麻烦

**Comment by Xiaoprinceprogrammer** [comment]
有时候扫码扫描第一次显示成功但是登录不了，得扫描两次。
还有生成的二维码怎么只有手表的 没有其他的吗。
还有现在戳一戳消息判断不出来，还有一些表情也判断不出来

**Comment by Xiaoprinceprogrammer** [comment]
这两个版本功能一样吗

**Comment by Xiaoprinceprogrammer** [comment]
![Uploading 360截图20220803170245273.jpg…]()

**Comment by Ink-33** [comment]
> ![Uploading 360截图20220803170245273.jpg…]()
> 

图没上传成功

**Comment by Mrs4s** [comment]
目前有`MacOS`协议 并不需要Windows协议
Windows协议和已有的实现差别太大

**Comment by Xiaoprinceprogrammer** [comment]
arm64 Windows | go-cqhttp_windows_arm64.exe | go-cqhttp_windows_arm64.zip
armv7 Windows | go-cqhttp_windows_armv7.exe | go-cqhttp_windows_armv7.zip
这两种功能一样吗

**Reply by Mrs4s** [reply]
一样 只是不同的目标架构而已

**Comment by Xiaoprinceprogrammer** [comment]
好 以后有问题 我在问你们   谢谢
